airbyte-connector-builder/.gitignore

```diff
@@ -0,0 +1,3 @@
+.coverage
+.venv
+state_*.yaml
```

Comment:
We probably want to add `connector_builder/generated` to this .gitignore, right? Since I don't think we want to be committing generated files to github, as that would require people to always commit the generator output to github